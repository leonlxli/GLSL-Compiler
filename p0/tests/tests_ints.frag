1000 +1000 -1000
0100 00000  -01
3u -3u -03u
3u0 3uu
u
1
+4
-3
2

3U
03U
0U
U