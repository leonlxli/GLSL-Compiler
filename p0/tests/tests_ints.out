1000         line 1 cols 1-4 is T_IntConstant (value = 1000)
+            line 1 cols 6-6 is '+' 
1000         line 1 cols 7-10 is T_IntConstant (value = 1000)
-            line 1 cols 12-12 is '-' 
1000         line 1 cols 13-16 is T_IntConstant (value = 1000)
0100         line 2 cols 1-4 is T_IntConstant (value = 100)
00000        line 2 cols 6-10 is T_IntConstant (value = 0)
-            line 2 cols 13-13 is '-' 
01           line 2 cols 14-15 is T_IntConstant (value = 1)
3u           line 3 cols 1-2 is T_IntConstant (value = 3)
-            line 3 cols 4-4 is '-' 
3u           line 3 cols 5-6 is T_IntConstant (value = 3)
-            line 3 cols 8-8 is '-' 
03u          line 3 cols 9-11 is T_IntConstant (value = 3)
3u           line 4 cols 1-2 is T_IntConstant (value = 3)
0            line 4 cols 3-3 is T_IntConstant (value = 0)
3u           line 4 cols 5-6 is T_IntConstant (value = 3)
u            line 4 cols 7-7 is T_Identifier 
u            line 5 cols 1-1 is T_Identifier 
1            line 6 cols 1-1 is T_IntConstant (value = 1)
+            line 7 cols 1-1 is '+' 
4            line 7 cols 2-2 is T_IntConstant (value = 4)
-            line 8 cols 1-1 is '-' 
3            line 8 cols 2-2 is T_IntConstant (value = 3)
2            line 9 cols 1-1 is T_IntConstant (value = 2)
3U           line 11 cols 1-2 is T_IntConstant (value = 3)
03U          line 12 cols 1-3 is T_IntConstant (value = 3)
0U           line 13 cols 1-2 is T_IntConstant (value = 0)
U            line 14 cols 1-1 is T_Identifier 
